pkg/backup/item_backupper.go

```diff
@@ -147,6 +147,12 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 		return nil
 	}
 
+	resource := obj.UnstructuredContent()
+	if deletionTimestamp, err := collections.GetString(resource, "metadata.deletionTimestamp"); err == nil {
```

Comment:
You can use `metadata.GetDeletionTimestamp()`

Reply:
Done.